packages/web/src/hooks/useVestedCoinsInfo.ts

```diff
@@ -0,0 +1,174 @@
+import {
```

Comment:
@rickyrombo if we can do all this work backend then we can avoid this hook and also don't have to install meteora and BN.js on web .